terraform/modules/task-definitions/router/main.tf

```diff
@@ -57,19 +55,19 @@ module "task_definition" {
           "awslogs-create-group" : "true",
           "awslogs-group" : "awslogs-fargate",
           "awslogs-region" : "eu-west-1",
-          "awslogs-stream-prefix" : "awslogs-${local.service_name}"
+          "awslogs-stream-prefix" : "awslogs-${var.service_name}"
         }
       },
       "mountPoints" : [],
       "portMappings" : [
         {
-          "containerPort" : 8080,
-          "hostPort" : 8080,
+          "containerPort" : 3054,
+          "hostPort" : 3054,
           "protocol" : "tcp"
         },
         {
-          "containerPort" : 8081,
-          "hostPort" : 8081,
+          "containerPort" : 3055,
+          "hostPort" : 3055,
```

Comment:
The Envoy proxy configuration for this task definition probably will need modifying to support these ports. At the moment (I believe) the sidecar proxy expects the app container to expose port 80 only.

No need to fix this in this PR, though!

Reply:
Thanks, I need to have a look how envoy is configured for apps.

Reply:
i fixed the envoy part i think but not the app mesh part because you can't have 2 listener per service: https://github.com/aws/aws-app-mesh-roadmap/issues/120